src/insexpand.c

```diff
@@ -1985,15 +1985,15 @@ ins_compl_len(void)
 }
 
 /*
- * Return TRUE when preinsert is set AND both 'menu' and 'menuone' flags
- * are also set, otherwise return FALSE.
+ * Return TRUE when "preinsert" is set AND the "menuone" flag is also set,
```

Comment:
Actually wait. This doesn't mention that `fuzzy` disables `preinsert`. It's probably better to just use this:
```suggestion
 * Return TRUE when the 'completeopt' "preinsert" flag is in effect,
```

Reply:
Okay more better